CHANGELOG.md

```diff
@@ -166,6 +166,8 @@
 
 - Go to definition now works for direct type annotations ([Ameen Radwan](https://github.com/Acepie))
 
+- Completions are now provided for values and types for use in unqualified imports ([Ameen Radwan](https://github.com/Acepie))
```

Comment:
This should work much better now. Didn't realize this was only working well because of auto bracket close before. lmk if this seems good enough now. If not I can change this

Reply:
I've just given this a try and it feels so nice! Thank you!!